### PR TITLE
new design mudb, solve concurrency.

### PR DESCRIPTION
**Pull request by MohsenNz**
check it out, thank you.

### CODE REVIEW

File: executor/src/mu_stack/deploy.rs [hunk 1]
```diff
@@ -1,11 +1,9 @@
-use db::MuDB;
 use reqwest::Url;
 use thiserror::Error;
-use tokio::task::spawn_blocking;
 
 use crate::{
     gateway::GatewayManager,
-    mudb::{client::DatabaseID, db},
+    mudb::service::{DatabaseID, Service as DbService},
```
Comment (Arshia001) [inline executor/src/mu_stack/deploy.rs:6]: I don't like how this import is aliased every time. If we need to rename it, why not just name it `DbService`? And while we're at it, I think `DatabaseManager` is a better name.

File: executor/src/mudb/service.rs [hunk 2]
```diff
@@ -1,138 +1,240 @@
...
-    table_name: String,
-) -> Result<CreateTableOutput> {
-    CreateTable {
-        database_id,
-        input: CreateTableInput { table_name },
+pub type Key = String;
+pub type Value = String;
+pub type Item = (Key, Value);
+
+#[derive(Debug, Clone)]
```
Comment (Arshia001) [inline executor/src/mudb/service.rs:15]: Why separate `Manager` and `Service`? The `Manager` could just as easily include all this functionality.

File: executor/src/mu_stack/deploy.rs [hunk 3]
```diff
@@ -1,11 +1,9 @@
...
     runtime::{
```
Comment (Arshia001) [inline executor/src/mu_stack/deploy.rs:7]: I don't like how this import is aliased every time. If we need to rename it, why not just name it `DbService`? And while we're at it, I think `DatabaseManager` is a better name.

**Review by Arshia001 — CHANGES_REQUESTED** [review]

File: executor/src/mudb/service.rs (hunk 2 above)
Comment (MohsenNz) [reply executor/src/mudb/service.rs:15]: .

Comment (MohsenNz) [reply executor/src/mudb/service.rs:15]: .

**Pull request merged**